generator-torsion/generators/app/templates/_d8/_cmm_torsion.libraries.yml

```diff
@@ -3,7 +3,7 @@ global:
   version: 1.x
   css:
     theme:
-      css/dist/style.css: { }
+      css/style.css: { }
```

Comment:
We don't need a /dist folder anymore since the /<theme_name> IS the destination folder right?